### ISSUE TITLE
Weather Sync Issue

### ISSUE THREAD

**Issue by FakeItalianMan** [Wyste/ServerSync#7]
So I have a bit of a problem when I set the weather and someone joins the server after I set the weather they don't see the weather I set. Any ideas on how to fix that?

**Comment by Wyste** [comment]
The weather won't be the same when they first join, but it should immediately transition to the set weather.

**Comment by FakeItalianMan** [comment]
It doesn't though I gotta reset it whenever someone joins. Unless I have to wait a while for it to transition?

**Comment by Wyste** [comment]
It should take about 1 real life minute to fully transition. By default the server cycles weather every 15 minutes, and it would sync them up shortly after that worst case.

I can't see how in a worst case scenario the approx 15 minutes is too long for weather to be out of sync?

**Comment by FakeItalianMan** [comment]
Ya my only problem is though is like right now I have it snowing in the server but when people join or I relog I load in and it will be clear weather and I have to set it to snow again for me or anyone else who joins after to see it when we load in. But it could have to do with how we set it up we extended how long the weather changes because it was changing too fast for us so that also could be the problem as well because it is taking longer for it to sync up to them? I'm not sure could be wrong.
